### DISCUSSION TITLE
multiple sessions management

### DISCUSSION THREAD

**Discussion by sagarshah1983** [ory/hydra#2331] [Q&A] [answered]
Assuming, that one user has already signed into the app/client using hydra OIDC solution and has not logged out. If another user now tries to login to the same app or different app from same browser, we see 2 possible options, but not sure, if this is possible or not.
- automatically kill previous session (perform logout by calling some hydra api) and seamlessly create new cookie with current user
- Provide user a logout link on an page suggesting to logout previous user session

Any suggestion on these approaches or any other alternative available with hydra?
Please advise.
Appreciate your inputs!

**Comment by vinckr** [comment, marked as answer]
The first option sounds like the most elegant imo.
But both should be possible.
Check the [spec for front-channel logout](https://openid.net/specs/openid-connect-frontchannel-1_0.html) and our [logout documentation](https://www.ory.sh/hydra/docs/concepts/logout/).

**Reply by sagarshah1983** [reply]
Thanks for sharing this links. That helps understanding logout flow. I think, we may be able to manage by just redirecting with login flow rejection from idp knowing that skip = true but sub field has a different value for another client.
